index.html

```diff
@@ -130,9 +136,10 @@ <h2>About me</h2><img class="right-corner" src="icons/Mask Group.svg" alt="">
           don’t hesitate to contact me.</p>
```

Comment:
- [ ] I kindly suggest that you follow project requirements by ensuring that your See Project buttons link to either the Live Demo of the featured project or to the GitHub repository.